js/slider.js

```diff
@@ -253,6 +345,10 @@
         this.$active = this.$slides.eq(this.activeIndex);
         let $caption = this.$active.find('.caption');
         this.$active.removeClass('active');
+        // Enables every slide
+        this.$slides.each((el) => {
+          el.style.visibility = 'visible';
+        });
```

Comment:
This is due to a characteristic of the transitons. For now, only the current slide should be "visible". So, before the transition is initiated, every slide must be set to visible in order not to result in a "blank area" being displayed. In other words, they are temporarily made available so as to avoid visual issues, which I've detected when testing it :)

Reply:
I've also updated it so as to use jQuery syntax in the last commit :)